src/HypothesisClient/ApiClient/ApiClientInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace eLife\HypothesisClient\ApiClient;
+
+use eLife\HypothesisClient\Credentials\CredentialsInterface;
+use eLife\HypothesisClient\HttpClient\HttpClientInterface;
+
+interface ApiClientInterface
+{
+    public function __construct(HttpClientInterface $httpClient, array $headers = []);
+
+    public function setCredentials(CredentialsInterface $credentials);
```

Comment:
Services shouldn't be stateful. This should be configuration.

Reply:
Some of the queries to the Hypothesis API can be done without credentials. Listing public annotations is one of them.